src/sentry/api/serializers/models/organization_member/base.py

```diff
@@ -72,7 +71,19 @@ def get_attrs(
         users_by_id: Mapping[str, Any] = {
             u["id"]: u for u in user_service.serialize_many(filter={"user_ids": users_set})
         }
-        actor_ids = [u.actor_id for u in user_service.get_many(filter={"user_ids": users_set})]
+        actor_ids = [
+            u.actor_id for u in user_service.get_many(filter={"user_ids": users_set}) if u.actor_id
+        ]
```

Comment:
We'll also need to cut out `user.actor_id` usage as that column is being removed. If we only need the actor ids for a set of users we could do 

```python
actors_ids = Actor.objects.filter(type=ACTOR_TYPES["user"], user_ids__in=users_set).values_list("id", flat=True)
```

To get the relevant actor records for the users in the current region.